c2corg_ui/templates/helpers/edit.html

```diff
@@ -168,7 +167,19 @@ <h3 class="heading show-phone" translate>summary</h3>
 </%def>
 
 <%def name="show_protected_warning(model)">\
-  <article class="protected-warning" ng-show="${model}.protected">
+  <article class="protected-warning" ng-if="${model}.protected">
     <span class="glyphicon glyphicon-warning-sign"></span>&nbsp;<span translate>This document is protected and can be edited by moderators only.</span>
   </article>
 </%def>
+
+<%def name="show_edit_framework(model)">\
+  <section class="section edit-framework" ng-if="!${model}.protected">
+    <h2 class="heading show-phone" translate>Framework of your contribution</h2>
+    <div class="content">
+      <div class="icon">
+        <span class="glyphicon glyphicon-warning-sign"></span>
+      </div>
+      <div class="details" translate>${model} edit framework details</div>
```

Comment:
If you use ``translate`` here, it will extract ``${model} edit framework details``. Which is probably not what you want. Then you have to use ``x-translate`` and add the strings explicitly to the ``i18n.html`` template.

But shouldn't you rather add those contents directly to the editing form templates without relying on a transifex "variable" ?

Reply:
Right, I forgot this for the `x-translate`.

About the variable, I would like to keep a single helper methd for all documents. How would you avoid a variable in the translation string?
Or maybe you see no interest in this factorization?

Reply:
According to your screenshot, the content of the translated variable will probably contain HTML tags such as list or links etc. Is Transifex really the place to write HTML tags? Shouldn't it only suggest explicit strings to translate?

Factorizing using an helper seems a good thing. Perhaps you could use a ``<%block>`` to override the content in each template?